app/src/main/java/com/github/fribourgsdp/radio/PlaylistSongsFragment.kt

```diff
@@ -33,6 +39,24 @@ class PlaylistSongsFragment : MyFragment(R.layout.fragment_playlist_display), On
         val playlistTitle : TextView = requireView().findViewById(R.id.PlaylistName)
         playlistTitle.text = playlistName
         initializeRecyclerView()
+        //sets listeners to 2 buttons
+        editButton = requireView().findViewById(R.id.editButton)
+        editButton.setOnClickListener {
+            val intent  = Intent(context, AddPlaylistActivity::class.java)
+            intent.putExtra(PLAYLIST_TO_MODIFY, Json.encodeToString(playlist))
+            startActivity(intent)
+        }
+        deleteButton = requireView().findViewById(R.id.deleteButton)
+        deleteButton.setOnClickListener{
```

Comment:
I think that clicking on a playlist to delete is rather unintuitive, even if indicated. It think it would be more intuitive to be able to slide it to the side in order to delete it.

Reply:
It may not be intuitive for you, but I think for many people of the "older" generation it is actually much more intuitive to first select the playlist and then delete it with the appropriate button. The risk with sliding is that people unintentionally delete their playlists.

One feature that could be added is the fact that when a user clicks on delete, a popup comes up and asks if he really wants to delete it. This double check could be a nice addition but is most definitely not needed for now.

Reply:
I see what you mean @BartLucien, i.e. sliding instead of clicking on _Songs_ to remove them, _once we are in the AddPlaylistActivity_ . This swiping could indeed be implemented.
The popup on playlist deletion is also a great idea, I will implement it in a future sprint.